environment.yml

```diff
@@ -14,3 +14,4 @@ dependencies:
     - twine
     - hatch
     - jupysql
+    - ploomber-core
```

Comment:
this is for the development environment. you need to edit: https://github.com/ploomber/jupysql-plugin/blob/2549029aad8e887c93c24926e8a21855b7fbf5be/pyproject.toml#L25